engine_comparison/begin_experiment.sh

```diff
@@ -25,30 +25,30 @@ else
 fi
 
 # Create one gcloud instance for the dispatcher
-gcloud compute instances create $INSTANCE_NAME --image-family=$DISPATCHER_IMAGE_FAMILY --image-project=$PROJECT_NAME
+gcloud compute instances create $INSTANCE_NAME --image-family=ubuntu-1604-lts --image-project=ubuntu-os-cloud
+
+gcloud compute ssh $INSTANCE_NAME --command="mkdir /input"
 
 # Send configs for the fuzzing engine
-export FENGINE_CONFIGS_DIR=${FENGINE_CONFIGS_DIR:-"~/fuzzing_engine_configs"}
 FENGINE_CONFIGS=${@:2}
 
 rm -rf ${SCRIPT_DIR}/tmp-configs
 mkdir ${SCRIPT_DIR}/tmp-configs
 for FENGINE in $FENGINE_CONFIGS; do
   cp $FENGINE tmp-configs/
```

Comment:
Are we making an assumption here about the CWD by referring to `tmp-configs` rather than `${SCRIPT_DIR}/tmp-configs`?